modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/WSDLImportTestCase.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's change the license header year

Reply:
Fixed with 60a7a8adb9c4542966f94c48dbed701f1e70945b